src/components/dls/Button/Button.module.scss

```diff
@@ -313,7 +313,7 @@
 }
 
 // when button used as Popover's trigger, add style when Popover is open
-[data-state="open"] .base {
+[data-state="open"] > button > .base {
```

Comment:
this fix the previous bug caused by #911 

The bug was :
- in repeat settings. the + button was highlighted.